src/commonMain/kotlin/fr/acinq/lightning/channel/Commitments.kt

```diff
@@ -123,7 +123,7 @@ sealed class LocalFundingStatus {
         override val fee: Satoshi = sharedTx.tx.fees
     }
 
-    data class ConfirmedFundingTx(override val signedTx: Transaction, override val fee: Satoshi) : LocalFundingStatus() {
+    data class ConfirmedFundingTx(override val signedTx: Transaction, override val fee: Satoshi, val localSigs: TxSignatures) : LocalFundingStatus() {
```

Comment:
It's a bit annoying that we have to store this here, but:

- it's annoying to recompute (because we could have wallet inputs)
- it's relatively small (a few hundreds of bytes at most)
- it makes the code trivially spec-compliant (always re-send `tx_signatures` if requested, regardless of confirmation state)

Reply:
Why don't we extract our sigs from `signedTx`?

Reply:
For the 2-of-2 it's easy, we just compare `local/remoteFundingPubkey`. However for our other inputs, I don't know...

Reply:
I think it's not worth the additional complexity to be honest, I don't see how we could simply and reliably figure out which inputs are ours.

Reply:
I agree, let's drop the idea.